contracts/test/LibLiquidationMock.sol

```diff
@@ -45,7 +46,7 @@ library LibLiquidationMock {
         uint256 maxSlippage,
         uint256 avgPrice,
         uint256 receiptPrice,
-        bool receiptSide
+        uint256 receiptSide
```

Comment:
any specific reason for doing this over just passing in a `Perpetuals.Side` enum?

Reply:
Yeah there was a type mismatch if I tried to make it `Perpetuals.Side`, but it worked if I casted inside the contracts